colorbleed/plugins/maya/publish/collect_renderlayers.py

```diff
@@ -122,6 +122,15 @@ def parse_options(self, render_globals):
 
         options = {"renderGlobals": {}}
         options["renderGlobals"]["Priority"] = attributes["priority"]
+
+        # Check for specific pools
+        pool_str = attributes.get("pools", None)
+        if pool_str is not None and not "":
```

Comment:
Wouldn't this check exactly the same if you check `if pool_str` :) Doesn't matter really.